packages/api-clients/open-api/bffApi.yml

```
@@ -82,15 +82,15 @@ tags:
       description: Find out more
       url: "http://swagger.io"
 paths:
-  "/agreements":
+  "/agreements/consumers":
```

For the e-services in this same file we use the pattern `"/producers/eservices"`, so I would do the same here and also remove the `s` since we are fixing getting agreements for ONE consumer (the caller)
```suggestion
  "/consumer/agreements":
```